### PR TITLE
feat: Use PRE-07 data for calculations

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* #625

### CODE REVIEW

File: source/databricks/tests/conftest.py [hunk 1]
```diff
@@ -23,10 +23,12 @@
 import subprocess
 from typing import Generator, Callable, Optional
 
+from delta import configure_spark_with_delta_pip
```
Comment (BjarkeMeier) [inline source/databricks/tests/conftest.py:26]: Do you have a link to a good resource on this one that I can read?

File: source/databricks/tests/integration/calculator/balance_fixing/test_get_enriched_time_series_points_df.py [hunk 2]
```diff
@@ -76,310 +76,310 @@ def factory(
...
-
-    # Act
-    actual = _get_enriched_time_series_points_df(
-        raw_time_series_points,
-        metering_point_period_df,
-        timestamp_factory(period_start),
-        timestamp_factory(period_end),
-    )
-    assert actual.count() == expected_number_of_rows
+# @pytest.mark.parametrize(
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/balance_fixing/test_get_enriched_time_series_points_df.py:79]: Shouldn't these tests be readded?

File: source/databricks/tests/integration/calculator/calculation_input/test_get_metering_point_periods_df.py [hunk 3]
```diff
@@ -125,81 +125,77 @@ def factory(
...
-            & (col("Type") == metering_point_type)
-            & (col("SettlementMethod") == settlement_method)
-            & (col("FromGridAreaCode") == "some-in-gride-area")
-            & (col("ToGridAreaCode") == "some-out-gride-area")
-            & (col("Resolution") == MeteringPointResolution.hour.value)
-            & (col("EnergySupplierId") == energy_supplier_id)
-        ).count()
-        == 1
-    )
+    # def test__when_metering_point_period_is_in_grid_areas__returns_metering_point_period(
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/calculation_input/test_get_metering_point_periods_df.py:128]: Re-add tests?

File: source/databricks/tests/integration/calculator/test_calculator_job.py [hunk 4]
```diff
@@ -50,7 +50,7 @@ def test_data_job_parameters(
         {
             "data_storage_account_name": "foo",
             "data_storage_account_key": "foo",
-            "wholesale_container_path": f"{test_files_folder_path}",
+            "wholesale_container_path": f"{data_lake_path}/delta/",
             "process_results_path": f"{data_lake_path}/{worker_id}/results",
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/test_calculator_job.py:54]: This seems a bit off compared to production as process results are stored in the wholesale container.

File: source/databricks/tests/integration/calculator/test_calculator_job.py [hunk 5]
```diff
@@ -436,7 +447,7 @@ def test__quarterly_basis_data_file_matches_contract(
         contract.extension,
     )
 
-    # Act: Executed in fixture executed_calculation_job
+    #     # Act: Executed in fixture executed_calculation_job
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/test_calculator_job.py:450]: Uncomment these 3 double commented assert comments?

File: source/databricks/package/balance_fixing.py [hunk 6]
```diff
@@ -38,7 +38,8 @@ def calculate_balance_fixing(
     period_end_datetime: datetime,
     time_zone: str,
 ) -> tuple[DataFrame, tuple[DataFrame, DataFrame], DataFrame]:
-
+    print("---------------------")
+    metering_points_periods_df.show()
```
Comment (BjarkeMeier) [inline source/databricks/package/balance_fixing.py:42]:
```suggestion
```

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**